### PR TITLE
Update to handle Cisco ISR mac address table  with multiple EHWIC mod…

### PR DESCRIPTION
**Pull request by pumpernickelBrad**
…ules installed

Example show output:

router#show mac-address-table
EHWIC: 0
Destination Address	Address Type	VLAN	Destination Port
-------------------	------------	----	-----------------
dca9.0476.1400		Dynamic      	  20	GigabitEthernet0/1/2
b8c1.1144.d7c6		Dynamic      	  19	GigabitEthernet0/1/2
bc16.65c1.fafe		Dynamic      	   1	GigabitEthernet0/3/2
988b.ad00.0209		Dynamic      	   1	GigabitEthernet0/1/3
e02f.6da3.505a		Dynamic      	   1	GigabitEthernet0/2/2
70ef.008c.4a53		Dynamic      	  14	GigabitEthernet0/1/2
0002.d162.1616		Dynamic      	  24	GigabitEthernet0/0/3
84b8.02ad.4e56		Dynamic      	   1	GigabitEthernet0/1/1
60c7.9834.2825		Dynamic      	  20	GigabitEthernet0/0/3
... <snip> ...

EHWIC: 1
Destination Address	Address Type	VLAN	Destination Port
-------------------	------------	----	-----------------
e098.6170.092c		Dynamic      	  22	GigabitEthernet0/1/2
18b4.30ae.4919		Dynamic      	  18	GigabitEthernet0/1/2
7c04.d0e9.5005		Dynamic      	  19	GigabitEthernet0/1/2
f81a.676f.94d2		Dynamic      	   1	GigabitEthernet0/1/3
6c41.6a36.5e16		Dynamic      	   1	GigabitEthernet0/0/3
609a.c1f1.9dc4		Dynamic      	  20	GigabitEthernet0/1/2
c8e0.eb9f.f8a3		Dynamic      	  22	GigabitEthernet0/1/2
... <snip> ...

### CODE REVIEW

**Comment by ktbyers** [comment]
@pumpernickelBrad Looks like you need to run `black .` from the base of the repository to reformat one of the files.

**Comment by pumpernickelBrad** [comment]
Thank you. I've run black and reformatted the file. All checks have passed now.

**Comment by mirceaulinic** [comment]
FYI: we are planning to release NAPALM 3.0.0 soon. If you'd like your patch to be included in this version, please address the comments above. Thanks!

**Comment by pumpernickelBrad** [comment]
Is there documentation for adding testcases? Looks like there are two places with mock data and they don't both have the same show commands in them.

File: test/ios/mocked_data/test_get_mac_address_table/isr_multiple_ehwic_format/expected_result.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	"mac": "11:11:22:22:33:33",
+	"interface": "GigabitEthernet0/0/1",
+	"moves": -1,
+	"last_move": -1.0
+},
+{
+	"static": false,
+	"active": true,
+	"vlan": 1,
+	"mac": "3333.2222.1111",
```
Comment (mirceaulinic) [inline test/ios/mocked_data/test_get_mac_address_table/isr_multiple_ehwic_format/expected_result.json:14]: I had the feeling I've already commented on this previously - looks like I didn't... Yes, you placed the mock data correctly, just a minor correction: https://travis-ci.org/github/napalm-automation/napalm/jobs/661437097#L513 so the MAC should be formatted as `33:33:22:22:11:11` on this line.

**Pull request merged**